packages/swapkit/thorchain/src/tcPlugin.ts

```diff
@@ -476,6 +477,7 @@ const plugin = ({ wallets, stagenet = false }: { wallets: Wallet; stagenet?: boo
               : chain === Chain.BinanceSmartChain
                 ? TCBscDepositABI
                 : TCEthereumVaultAbi;
+          // TODO Arbitrum ABI?
```

Comment:
@chillios-dev do you know if we need a extra ABI for Arbitrum? or is the Ethereum ABI enough?